core/command_handler.py

```
@@ -0,0 +1,181 @@
+        async with self._hold_or_create_lock():
+            await self.queue.put(
+                (
+                    priority_int,  # Priority to sort commands with
+                    next(self.cmd_counter),  # A counter to serve as a tie-breaker
+                    deepcopy(cmd_data),  # actual data
+                )
+            )
+            self.cmds_state[cmd_data["id"]]["in_queue"] = True
+
+    @suppress_and_log("Remove Queue")
+    async def remove_queue(self, cmd_data=None, id=None):
```

Ruff (0.16.0)

[error] Function argument `id` is shadowing a Python builtin

(A002)